docs/sources/configure-client/language-sdks/go_push.md

```diff
@@ -150,11 +160,37 @@ pyroscope.Start(pyroscope.Config{
 })
 ```
 
-To configure the Golang SDK to send data to Pyroscope, replace the `<URL>` placeholder with the appropriate server URL. This could be the Grafana Cloud URL or your own custom Pyroscope server URL.
+### Option: Use `DisableGCRuns` for handling increased memory usage
+
+Pyroscope may require additional when tracking a lot of objects. For example, a Go service that indexes large amounts of data requires more memory.
+This tracking can lead to higher CPU usage and potential CPU throttling.
+
+You can use `DisableGCRuns` in your Go configuration to disable automatic runtimes.
+If this flag is activated, there is less GC running and therefore less CPU resources spent.
+However, the heap profile may be less precise.
+
+#### Background
```

Comment:
@kolesnikovae, I've incorporated your content into a new section. What do you think? maybe a different heading title besides background?

Reply:
Everything looks good!

One point that could be clarified is the title. Currently, it says "use ... for handling increased memory usage", which might suggest this is specifically about the profiler/SDK (which would be wrong, as it mainly affects the CPU usage). I'd suggest revising the title to:

```md
### Option: Use `DisableGCRuns` for memory-intensive applications
```

The "Background" title works well, but I’m not entirely sure a separate title is needed for that paragraph.